### PR TITLE
Zy26

### PR DESCRIPTION
**Pull request by YZY000403**
current price works better using yahoo_fin instead of yfinance

### CODE REVIEW

**Review by avmehta2 — APPROVED** [review]

File: Modelling/test array.py [hunk 1]
```diff
@@ -0,0 +1 @@
+dfgsfsdfdsf
```
Comment (MattGeimer) [inline Modelling/test array.py:1]: What is this file for?

File: Modelling/HelperTicker.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+def gethistoricalprice(tickers):
+    return yf.history(tickers, period="max")
+
+
+def getcurrentprice(tickers):
+    return si.get_live_price(tickers)
+
+
+# print(getcurrentprice('AMZN'))
```
Comment (MattGeimer) [inline Modelling/HelperTicker.py]: This late in the timeline, code should not contain commented out lines. They should either be removed completely or uncommented.

File: Modelling/yourpage.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+# var2.set(('stock a', 'stock b', 'stock c', 'stock d'))
+lb = tk.Listbox(window, listvariable=var2, width = 46)
+
+# list_items=[1,2,3,4]
+# for item in list_items:
+#   lb.insert('end',item)
+# lb.insert(1, 'stock e')
+# lb.insert(2, 'stock f')
+# lb.insert(3, 'stock g')
+# lb.insert(4, 'stock h')
```
Comment (MattGeimer) [inline Modelling/yourpage.py]: This late in the timeline, code should not contain commented out lines. They should either be removed completely or uncommented.

**Review by MattGeimer — COMMENTED** [review]
Please make sure this code is representative of final product work. There should not be commented out code. Also, `test array.py` appears to not be a real file and should be removed prior to being pulled.

**Comment by MattGeimer** [comment]
@YZY000403 Please fix the issues ASAP so that I can merge this and re-structure the repo.

**Review by MattGeimer — COMMENTED** [review]
Can you remove test array.py

**Pull request closed without merging**